src/Documentarian.php

```diff
@@ -92,7 +92,8 @@ public function generate($folder)
 
         $output = $renderer->render('index', [
             'page' => $frontmatter,
-            'content' => $html
+            'content' => $html,
+            'folder' => str_replace('public', null, $folder)
```

Comment:
Shouldn't you be replacing `/public`?

Reply:
Tests are failing. It seems you didn't update the expected generated docs with the new links. (Which is going to be difficult, since the folder path will vary)